integrations/outlook/src/definitions/index.ts

```diff
@@ -0,0 +1,50 @@
+import type { IntegrationDefinitionProps } from "@botpress/sdk";
+import { z } from "zod";
+
+export { channels } from "./channels";
+
+export const configuration = {
+  schema: z.object({
+    tenantId: z.string({
+      description: "Azure AD tenant id",
+    }),
```

Comment:
`z.string().describe()` is a bit clearer and makes the schema smaller

Reply:
Yo @allardy, I try to use conventional comments when reviewing a PR. This is a **nitpick**